stac-gen/handler.py

```diff
@@ -0,0 +1,84 @@
+from cmr import GranuleQuery
+import os
+import json
+import pystac
+from pystac.utils import str_to_datetime
+from shapely.geometry import shape
+from pypgstac import pypgstac
+from rio_stac.stac import bbox_to_geom, create_stac_item
+
+HOST = os.environ.get('HOST')
+USER = os.environ.get('USER')
+PASSWORD = os.environ.get('PASSWORD')
+
+def create_item(cmr, cog, collection):
+
+    assets = {}
+
+    for link in cmr["links"]:
+        if ".he5" in link["href"]:
+            name = link["title"] if "title" in link else link["href"]
+            assets[name] = pystac.Asset(
+                href=link["href"],
+                media_type="application/x-hdf5",
+                roles=["data"],
+                title="hdf image",
+            )
+
+    dt = str_to_datetime(cmr["time_start"])
+
+    try:
+        rstac = create_stac_item(
+            source=cog,
+            collection=collection,
+            input_datetime=dt,
+            properties=cmr,
+            with_proj=True,
+            with_raster=True,
+            assets=assets,
```

Comment:
If I'm reading this code correctly, `assets` is an object with only one key/value pair for the HDF file, I think we also want the COG itself to be an asset.

Reply:
added